test/config.mjs

```diff
@@ -385,6 +385,17 @@ describe("configuration file", function () {
       expect(stripAnsi(stderr.getWritten())).to.equal(message);
     });
 
+    it("exits with an error if a profile is specified and there is no config", async function () {
+      try {
+        await run(`eval "Database.all()" --profile nonexistent`, container);
+      } catch (e) {}
+
+      const errorText = `Profile "nonexistent" cannot be specified, because no config file found at "${path.join(__dirname, "..")}". Remove the profile, or provide a config file.`;
```

Comment:
```suggestion
      const errorText = `Profile "nonexistent" cannot be specified because there's no config file found at "${path.join(__dirname, "..")}". Remove the profile, or provide a config file.`;
```